local/client.go

```diff
@@ -73,6 +75,19 @@ func WithTLS(tlsConfig *TLSConfig) OptionFunc {
 	}
 }
 
+type DiscoveryConfiguration struct {
+	MulticastHopLimit    int
```

Comment:
Please add a comment about what the default is and how to set the unlimited number of hops.